app/lib/event_store/event.rb

```diff
@@ -84,7 +84,7 @@ def initialize(error:, event:)
     # The problem is that this triggers the Event's `load` and `deserialize` (to restore the original state), and if
     # the object referenced in the event via GlobalID does not exist, the deserialization of the event will fail.
     # Prevent an exception during rollback but try to report when that's unsafe.
-    def rolledback!(*)
+    def rolledback!(...)
```

Comment:
🤯

Reply:
same here, doesn't `*` continue to work well?

Reply:
I don't remember the error and how to reproduce it, but this method accepts keyword arguments, so I think `*` wouldn't work: https://github.com/rails/rails/blob/v6.1.7.8/activerecord/lib/active_record/transactions.rb#L329

Reply:
The tests fail with `*`, I am a bit lazy to dig why exactly, because I probably did that already before, but in general I think `...` is safer for any further method definition changes: https://app.circleci.com/pipelines/github/3scale/porta/29173/workflows/e5d0f7f4-1cae-4682-a632-ee10693950e2/jobs/326772/tests